source/renderer/app/components/staking/stake-pools/StakePools.tsx

```diff
@@ -119,29 +119,47 @@ class StakePools extends Component<Props, State> {
     this.setState({
       search,
     });
+
   handleClearSearch = () =>
     this.setState({
       search: '',
     });
-  handleGridView = () =>
+
+  handleGridView = () => {
     this.setState({
       isGridView: true,
       isGridRewardsView: false,
       isListView: false,
     });
+
+    this.props.analyticsClient.sendEvent(
+      'Stake Pools',
```

Comment:
Should we make these event names as const so its better/safer to reuse?

Reply:
I agree. Same for `pageTitle`.

Reply:
Ok, pushing this change in https://github.com/input-output-hk/daedalus/pull/3003

Reply:
Got it. I think it's basically the same story as here: https://github.com/input-output-hk/daedalus/pull/3003#discussion_r913620638 - whatever we agree should be applied consistently.